src/solvers/defaults.jl

```diff
@@ -23,9 +21,9 @@ function default_region_printer(;
     region = first(sweep_plan[which_region_update])
     @printf("Sweep %d, region=%s \n", which_sweep, region)
     print("  Truncated using")
-    @printf(" cutoff=%.1E", cutoff)
-    @printf(" maxdim=%d", maxdim)
-    @printf(" mindim=%d", mindim)
+    haskey(inserter_kwargs, :cutoff) && @printf(" cutoff=%.1E", inserter_kwargs[:cutoff])
+    haskey(inserter_kwargs, :maxdim) && @printf(" maxdim=%d", inserter_kwargs[:maxdim])
+    haskey(inserter_kwargs, :mindim) && @printf(" mindim=%d", inserter_kwargs[:mindim])
```

Comment:
```suggestion
    haskey(inserter_kwargs, :cutoff) && @printf(" cutoff=%.1E", inserter_kwargs.cutoff)
    haskey(inserter_kwargs, :maxdim) && @printf(" maxdim=%d", inserter_kwargs.maxdim)
    haskey(inserter_kwargs, :mindim) && @printf(" mindim=%d", inserter_kwargs.mindim)
```
Just a style preference, I think it looks cleaner.

Reply:
Agreed – I forgot about that syntax which is much nicer